lib/salus/report.rb

```diff
@@ -158,17 +181,24 @@ def to_s(verbose: false, wrap: WRAP, use_colors: false)
     end
 
     def to_yaml
+      YAML.dump(to_h.deep_sort)
+    rescue StandardError => e
+      bugsnag_notify(e.inspect + "\n" + e.message + "\nResult String: " + to_h.to_s)
       YAML.dump(to_h)
     end
 
     def to_json
+      JSON.pretty_generate(to_h.deep_sort)
+    rescue StandardError => e
+      bugsnag_notify(e.inspect + "\n" + e.message + "\nResult String: " + to_h.to_s)
       JSON.pretty_generate(to_h)
     end
 
     def to_sarif(config = {})
       sarif_json = Sarif::SarifReport.new(@scan_reports, config, @repo_path).to_sarif
       # We will validate to ensure the applied filter
       # doesn't produce any invalid SARIF
+      sarif_json = JSON.pretty_generate(JSON.parse(sarif_json).deep_sort)
```

Comment:
wrap `deep_sort`  in `begin ... rescue ...`

Reply:
Please find all occurrences of `deep_sort` that are not wrapped in `begin ... rescue ...` and wrap.